### PR TITLE
Create global CONTRIBUTING doc | closes issue #15

### PR DESCRIPTION
**Pull request by khdc-me**
Changed text to be more general in nature.

Please take a look at step 3 under "Get Started" section.
I think the forking, cloning, rebasing info is important. And given that projects could have their own special way of running and might require a different set up than others, it might still be a good idea to keep those steps local to each project's repo.

Let me know if you still think that whole section should just be removed.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+4. Create a branch for local development:
+
+   .. code-block:: console
+
+        $ git checkout -b <name-of-your-bugfix-or-feature>
+
+   Now you can make your changes locally.
+
+5. When you're done making changes, check that your changes pass flake8 and the tests, including testing other Python versions with tox:
```
Comment (ovv) [inline CONTRIBUTING.rst]: `tox` is python specific and not used in all pyslackers projects, this point should probably just be `run the test to make sure everything is ok`.

**Review by ovv — CHANGES_REQUESTED** [review]
:+1:

File: CONTRIBUTING.rst (hunk 1 above)
Comment (khdc-me) [reply CONTRIBUTING.rst]: So leave open to use any test suite?

What about referring devs to project's contrb doc as in step 3? Maybe setting up the local contributing docs as:

## PySlackers' Contributing Guidelines
link to community/CONTRIBUTING.rst

## Running Dev Environment Locally
1) Steps
2) to set up
3) the environment

## How to test
This project uses pytest/tox/whatever for its unit tests.
1) Steps
2) to set up
3) and run test
4) w/ specific suite

Comment (ovv) [reply CONTRIBUTING.rst]: yeah exactly :+1:

**Comment by khdc-me** [comment]
Resubmitted, referring contributor to projects' local CONTRIBUTING file for steps on how to test.

**Review by ovv — COMMENTED** [review]
:+1:

**Review by ovv — DISMISSED** [review]

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,126 @@
+============
+Contributing
+============
+
+First off, thanks for taking the time to contribute!
+
+Contributions are welcome by anybody and everybody. We are not kidding! Looking for help ? Join us on `Slack`_ by requesting an `invite`_.
```
Comment (NDevox) [inline CONTRIBUTING.rst:7]: ~Random underscore at the end?~

~Also probably worth linking to the invite page.~

Comment (NDevox) [reply CONTRIBUTING.rst:7]: Ok I'm just not familiar with RST enough ignore me

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+If you are proposing a feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that these are volunteer-driven projects, and that contributions are welcome :)
+
+Get Started!
+------------
+
+Ready to contribute? Here is a our preferred approach to forking our projects:
```
Comment (NDevox) [inline CONTRIBUTING.rst]: `a our` -> `our`

**Comment by NDevox** [comment]
Singular typo is all for me. Then can merge.

**Comment by khdc-me** [comment]
The exact same thing happened to me when I first took at look at the source file, @NDevox. Took me a couple of mins to see that was all for constructing links.

Good catch on 'a our'.

**Comment by NDevox** [comment]
😉 LGTM

**Review by NDevox — APPROVED** [review]

**Pull request merged**